Tests/SmokeTestNoRealms/README.md

```diff
@@ -0,0 +1,23 @@
+# About SmokeTestNoRealms
```

Comment:
What's the point of this one? We should have weaver tests to cover the case, no need for a new project.